evm_loader/lib/src/config.rs

```diff
@@ -76,8 +98,22 @@ pub fn load_api_config_from_environment() -> APIOptions {
     }
 }
 
-/// # Errors
-fn load_db_config_from_environment() -> ChDbConfig {
+#[must_use]
+pub fn load_db_config_from_environment() -> DbConfig {
+    env::var("TRACER_DB_TYPE")
+        .ok()
+        .and_then(|db_type| match db_type.to_lowercase().as_str() {
+            "rocksdb" => Some(DbConfig::RocksDbConfig(
+                load_rocks_db_config_from_environment(),
+            )),
+            "clickhouse" => Some(DbConfig::ChDbConfig(load_ch_db_config_from_environment())),
+            "none" => Some(DbConfig::NoDbConfig(NoDbConfig {})),
```

Comment:
Make `TRACER_DB_TYPE` optional with `none` value by default.

Reply:
Now `TRACER_DB_TYPE` can be either not set or set to 'clickhouse', 'rocksdb' or 'none'.